cli/command_server_start.go

```diff
@@ -104,6 +109,7 @@ func (c *commandServerStart) setup(svc advancedAppServices, parent commandParent
 	cmd.Flag("ui-preferences-file", "Path to JSON file storing UI preferences").StringVar(&c.uiPreferencesFile)
 
 	cmd.Flag("log-server-requests", "Log server requests").Hidden().BoolVar(&c.logServerRequests)
+	cmd.Flag("disable-csrf-token-checks", "Disable CSRF token").Hidden().BoolVar(&c.disableCSRFTokenChecks)
```

Comment:
Wondering which approach would be better: "disabling CSRF checks in tests" vs "test-api to generate a CSRF token" for the test calls to the server ?

Right now it seems disabling is okay. Will think more about this ...

Reply:
Disabling is only used when developing the UI in a dev server using a proxy, so very rare.